examples/remote_procedure/mnist/run_task.py

```diff
@@ -52,7 +52,7 @@ def validate_screening_unit(unit: Unit):
     agent = unit.get_assigned_agent()
     if agent is not None:
         data = agent.state.get_data()
-        annotation = data["outputs"]["final_submission"]["annotations"][0]
+        annotation = data["final_submission"]["annotations"][0]
```

Comment:
You should update the front-end to index into outputs rather than this, such that we can have a more standard format/expectations on where to find data across different `AgentState` types: #1065